frontend/models/product-list/index.ts

```diff
@@ -74,6 +82,48 @@ export async function findProductList(
    };
 }
 
+type DeviceWiki = Record<string, any>;
+
+async function fetchDeviceWiki(
+   deviceTitle: string
+): Promise<DeviceWiki | null> {
+   const deviceHandle = getDeviceHandle(deviceTitle);
+   try {
+      const response = await fetch(
+         `${IFIXIT_ORIGIN}/api/2.0/wikis/CATEGORY/${deviceHandle}`
+      );
+      const payload = await response.json();
+      return payload;
+   } catch (error) {
+      return null;
+   }
+}
+
+function getDeviceImage(deviceWiki: DeviceWiki): ProductListImage | null {
+   return deviceWiki.image?.original == null
+      ? null
+      : {
+           url: deviceWiki.image.original,
```

Comment:
The `original` size is the largest version of the image that we have. If we can get away with it, we probably want to use the smallest size that we can without having to scale the image up in any view. I would guess that's one of `standard`, `medium`, or `large`, going from smallest to largest.

Reply:
@sctice-ifixit I'm using this because the resize is done by Next Image.. `original` is not the image that ends up being used client side, just the one used by Next.js to create the image set